uint8  servo_id
int16  present_current
int32  present_velocity
int32  present_position

